### PR TITLE
Class Comp -> FC -- IngredientList

### PR DESCRIPTION
**Pull request by rtragle**
This PR changes the IngredientList component from class based to functional w/ hooks

IngredientList FC refactor-2020-11-17_13 50 34

https://app.clickup.com/t/g6vu77

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -3751,26 +3751,6 @@
         }
       }
     },
-    "@testing-library/user-event": {
```
Comment (ayan4m1) [inline package-lock.json:3754]: Why is package-lock.json changing without a change to package.json?

**Review by ayan4m1 — APPROVED** [review]
Looks good, thanks! 🚀

File: package-lock.json (hunk 1 above)
Comment (rtragle) [reply package-lock.json:3754]: I was wondering the same thing... Do you want me to remove this change from the PR?

**Review by ayan4m1 — APPROVED** [review]
Thanks for making those changes!

**Pull request merged**